dht-node/src/dht_node/index.ts

```diff
@@ -4,10 +4,15 @@ import DHT from '@hyperswarm/dht'
 import { logger } from '@/server/logger'
 import CONFIG from '@/config'
 import { FederatedCommunity as DbFederatedCommunity } from '@entity/FederatedCommunity'
+import { Community as DbCommunity } from '@entity/Community'
+import { v4 as uuidv4 } from 'uuid'
 
 const KEY_SECRET_SEEDBYTES = 32
-const getSeed = (): Buffer | null =>
-  CONFIG.FEDERATION_DHT_SEED ? Buffer.alloc(KEY_SECRET_SEEDBYTES, CONFIG.FEDERATION_DHT_SEED) : null
+const getSeed = (): Buffer | null => {
+  return CONFIG.FEDERATION_DHT_SEED
+    ? Buffer.alloc(KEY_SECRET_SEEDBYTES, CONFIG.FEDERATION_DHT_SEED)
+    : null
+}
```

Comment:
yes it could, but this is the one and only place, where the seed is used to generate the keypair. So why do you want to shift code in a general used part like the config?